app/src/main/java/org/mozilla/fenix/home/recentbookmarks/view/RecentBookmarks.kt

```diff
@@ -91,40 +101,58 @@ private fun RecentBookmarkItem(
 ) {
     var isMenuExpanded by remember { mutableStateOf(false) }
 
-    Column(
+    Card(
         modifier = Modifier
-            .width(156.dp)
+            .width(158.dp)
             .combinedClickable(
                 enabled = true,
                 onClick = { onRecentBookmarkClick(bookmark) },
                 onLongClick = { isMenuExpanded = true }
-            )
+            ),
+        shape = cardShape,
+        backgroundColor = FirefoxTheme.colors.layer2,
+        elevation = 6.dp,
     ) {
-        Card(
+        Column(
             modifier = Modifier
                 .fillMaxWidth()
-                .height(96.dp),
-            elevation = 6.dp
+                .padding(horizontal = 16.dp)
         ) {
-            RecentBookmarkImage(bookmark)
-        }
+            Spacer(modifier = Modifier.height(16.dp))
 
-        Spacer(modifier = Modifier.height(8.dp))
+            Box(
+                modifier = imageModifier.then(
+                    Modifier.background(
+                        color = when (isSystemInDarkTheme()) {
+                            true -> PhotonColors.DarkGrey30
+                            false -> PhotonColors.LightGrey30
+                        },
```

Comment:
Is there an appropriate color variable to use here?

Reply:
Unfortunately, there isn't currently :(. The dark mode color will be separately updated in https://github.com/mozilla-mobile/fenix/issues/26609, but even after the update, there is no token matching the color usage here.